### PR TITLE
chore: switch wakuv2 to waku fleet

### PR DESCRIPTION
**Pull request by yakimant**

### CODE REVIEW

File: docs/contributors/continuous-integration.md [hunk 1]
```diff
@@ -10,10 +10,8 @@ https://ci.infra.status.im/job/nim-waku/
 
 It currently consists of four jobs:
 
-* [manual](https://ci.infra.status.im/job/nim-waku/job/manual/) - For manually executing builds using parameters.
```
Comment (Ivansete-status) [inline docs/contributors/continuous-integration.md:13]: I have the impression that the `manual` job is still valid.

**Review by Ivansete-status — APPROVED** [review]
LGTM!

File: docs/contributors/continuous-integration.md (hunk 1 above)
Comment (yakimant) [reply docs/contributors/continuous-integration.md:13]: Good catch, thank you!

File: apps/chat2/chat2.nim [hunk 2]
```diff
@@ -381,10 +381,10 @@ proc processInput(rfd: AsyncFD, rng: ref HmacDrbgContext) {.async.} =
     echo "Connecting to " & $conf.fleet & " fleet using DNS discovery..."
 
     if conf.fleet == Fleet.test:
-      dnsDiscoveryUrl = some("enrtree://AO47IDOLBKH72HIZZOXQP6NMRESAN7CHYWIBNXDXWRJRZWLODKII6@test.wakuv2.nodes.status.im")
+      dnsDiscoveryUrl = some("enrtree://AOGYWMBYOUIMOENHXCHILPKY3ZRFEULMFI4DOM442QSZ73TT2A7VI@test.waku.nodes.status.im")
     else:
       # Connect to prod by default
-      dnsDiscoveryUrl = some("enrtree://ANEDLO25QVUGJOUTQFRYKWX6P4Z4GKVESBMHML7DZ6YK4LGS5FC5O@prod.wakuv2.nodes.status.im")
+      dnsDiscoveryUrl = some("enrtree://AIRVQ5DDA4FFWLRBCHJWUWOO6X6S4ZTZ5B667LQ6AJU6PEYDLRD5O@sandbox.waku.nodes.status.im")
```
Comment (Ivansete-status) [inline apps/chat2/chat2.nim:387]: nitpick. We may need to also update the previous comment: "Connect to sandbox.."

File: examples/publisher.nim [hunk 3]
```diff
@@ -21,9 +21,9 @@ import
 proc now*(): Timestamp =
   getNanosecondTime(getTime().toUnixFloat())
 
-# An accesible bootstrap node. See wakuv2.prod fleets.status.im
-
+# An accesible bootstrap node. See waku.sandbox fleets.status.im
 
+#TODO: which host is this ENR for?
```
Comment (Ivansete-status) [inline examples/publisher.nim:26]: Sorry I can't quite understand the question

**Review by Ivansete-status — APPROVED** [review]
LGTM! Great work! :100:

File: docs/tutorial/dingpu.md [hunk 4]
```diff
@@ -25,9 +25,7 @@ There is also an interactive mode. Type `/connect` then paste address of other n
 
 ## Dingpu cluster node
 
-```
-/ip4/134.209.139.210/tcp/60000/p2p/16Uiu2HAmJb2e28qLXxT5kZxVUUoJt72EMzNGXB47Rxx5hw3q4YjS
-```
+> TODO (2024-03-11): Fix node multiaddr
```
Comment (SionoiS) [inline docs/tutorial/dingpu.md:28]: TODO but not in this PR I assume?

Same elsewhere

File: docs/tutorial/onchain-rln-relay-chat2.md [hunk 5]
```diff
@@ -79,11 +79,11 @@ then you will see a couple of other messages related to setting up the connectio
 the content may differ on your screen though:
 ```
 Connecting to test fleet using DNS discovery...
-Discovered and connecting to @[16Uiu2HAkvWiyFsgRhuJEb9JfjYxEkoHLgnUQmr1N5mKWnYjxYRVm, 16Uiu2HAmPLe7Mzm8TsYUubgCAW1aJoeFScxrLj8ppHFivPo97bUZ, 16Uiu2HAmJb2e28qLXxT5kZxVUUoJt72EMzNGXB47Rxx5hw3q4YjS]
+Discovered and connecting to @[16Uiu2HAkzHaTP5JsUwfR9NR8Rj9HC24puS6ocaU8wze4QrXr9iXp, 16Uiu2HAkykgaECHswi3YKJ5dMLbq2kPVCo89fcyTd38UcQD6ej5W, 16Uiu2HAkzHaTP5JsUwfR9NR8Rj9HC24puS6ocaU8wze4QrXr9iXp]
```
Comment (SionoiS) [inline docs/tutorial/onchain-rln-relay-chat2.md:82]: 1e and last are the same. Is it intended?

Same elsewhere

File: docs/tutorial/rln-chat2-live-testnet.md [hunk 6]
```diff
@@ -82,11 +82,11 @@ Bob
 Choose a nickname >> Bob
 Welcome, Bob!
 Connecting to test fleet using DNS discovery...
-Discovered and connecting to @[16Uiu2HAmPLe7Mzm8TsYUubgCAW1aJoeFScxrLj8ppHFivPo97bUZ, 16Uiu2HAkvWiyFsgRhuJEb9JfjYxEkoHLgnUQmr1N5mKWnYjxYRVm, 16Uiu2HAmJb2e28qLXxT5kZxVUUoJt72EMzNGXB47Rxx5hw3q4YjS]
+Discovered and connecting to @[16Uiu2HAkykgaECHswi3YKJ5dMLbq2kPVCo89fcyTd38UcQD6ej5W, 16Uiu2HAkzHaTP5JsUwfR9NR8Rj9HC24puS6ocaU8wze4QrXr9iXp, 16Uiu2HAkzHaTP5JsUwfR9NR8Rj9HC24puS6ocaU8wze4QrXr9iXp]
```
Comment (SionoiS) [inline docs/tutorial/rln-chat2-live-testnet.md:85]: 2e & 3e are the same.

Same elsewhere

**Review by SionoiS — APPROVED** [review]
Very nice thanks!

Left some comments.

**Review by NagyZoltanPeter — APPROVED** [review]
Approve with little nitpick.
..and a question: While we have this https://fleets.status.im/ but wouldn't it be possible to store all these hard coded addresses in a git repository with some script could be used to retrieve the necessary set of addresses and store in env variables to be used within our apps?
Such solution would give flexibility and automation for using the necessary fleet without such hard coding.
WDYT? cc:@SionoiS @Ivansete-status @gabrielmer @jm-clius

File: examples/publisher.nim (hunk 3 above)
Comment (yakimant) [reply examples/publisher.nim:26]: I just checked with https://enr-viewer.com, it is for `node-01.ac-cn-hongkong-c.wakuv2.prod` but peer id changed. Let me replace it with `node-01.ac-cn-hongkong-c.waku.sandbox` one:
```
enr:-QEkuEB3WHNS-xA3RDpfu9A2Qycr3bN3u7VoArMEiDIFZJ66F1EB3d4wxZN1hcdcOX-RfuXB-MQauhJGQbpz3qUofOtLAYJpZIJ2NIJpcIQI2SVcim11bHRpYWRkcnO4bgA0Ni9ub2RlLTAxLmFjLWNuLWhvbmdrb25nLWMud2FrdS5zYW5kYm94LnN0YXR1cy5pbQZ2XwA2Ni9ub2RlLTAxLmFjLWNuLWhvbmdrb25nLWMud2FrdS5zYW5kYm94LnN0YXR1cy5pbQYfQN4DgnJzkwABCAAAAAEAAgADAAQABQAGAAeJc2VjcDI1NmsxoQPK35Nnz0cWUtSAhBp7zvHEhyU_AqeQUlqzLiLxfP2L4oN0Y3CCdl-DdWRwgiMohXdha3UyDw
```

File: .github/ISSUE_TEMPLATE/prepare_release.md [hunk 7]
```diff
@@ -23,5 +23,5 @@ For detailed info on the release process refer to https://github.com/waku-org/nw
 - [ ] assign release tag to the cherry-picked release notes commit
 - [ ] create GitHub release
 - [ ] deploy the release to DockerHub
-- [ ] deploy release to `wakuv2.prod` fleet
+- [ ] deploy release to `waku.sandbox` fleet
```
[the remark this answers is not in the document]
Comment (SionoiS) [reply .github/ISSUE_TEMPLATE/prepare_release.md:26]: > Yes, as I see you also know this now from the discord channel.
> 
> This is the table - https://fleets.waku.org, what's the best place for it?

https://github.com/waku-org/nwaku/blob/master/docs/contributors/waku-fleets.md

**Comment by yakimant** [comment]
@NagyZoltanPeter, I admire your idea to extract hardcoded links and use https://fleets.status.im

Status Desktop and Mobile already do it. They store a copy of json in repo and update it with make target.

But this is out of the scope of this PR, I just want to replace the old references.
Although this PR can be useful for future work.

**Comment by yakimant** [comment]
@SionoiS, @NagyZoltanPeter, @Ivansete-status, some CI jobs are failing.
Was not trivial to figure out, what happened, can you plase help?

**Comment by SionoiS** [comment]
> @SionoiS, @NagyZoltanPeter, @Ivansete-status, some CI jobs are failing. Was not trivial to figure out, what happened, can you plase help?

At first glace -> `/home/runner/work/nwaku/nwaku/examples/publisher.nim(33, 3) Error: closing " expected`

**Comment by NagyZoltanPeter** [comment]
> > @SionoiS, @NagyZoltanPeter, @Ivansete-status, some CI jobs are failing. Was not trivial to figure out, what happened, can you plase help?
> 
> At first glace -> `/home/runner/work/nwaku/nwaku/examples/publisher.nim(33, 3) Error: closing " expected`

@yakimant: Yes, @SionoiS is right. In commit: b609ec570c3 there is a missing closing apostrophe.  
But I see there are other issues with it.

First of all it will be hard to merge in this way due to the nph formatting recently applied to the complete nwaku nim code base. 
I can help with it if you need!

**Comment by NagyZoltanPeter** [comment]
> > > @SionoiS, @NagyZoltanPeter, @Ivansete-status, some CI jobs are failing. Was not trivial to figure out, what happened, can you plase help?
> > 
> > 
> > At first glace -> `/home/runner/work/nwaku/nwaku/examples/publisher.nim(33, 3) Error: closing " expected`
> 
> @yakimant: Yes, @SionoiS is right. In commit: [b609ec5](https://github.com/waku-org/nwaku/commit/b609ec570c33a006a0e3d2b36ebb07ecf1475adc) there is a missing closing apostrophe. But I see there are other issues with it.
> 
> First of all it will be hard to merge in this way due to the nph formatting recently applied to the complete nwaku nim code base. I can help with it if you need!

@yakimant: Did you start a rebase to latest origin/master? If so it has not been finished.
I fixed the code, will push it here to see if ci will go well.

**Comment by NagyZoltanPeter** [comment]
@yakimant, @SionoiS : I pushed fixes for the code and test. Tests seems running well... but not perfect.
ubuntu-latest-rln-v2 fails with port is already occupied that test would use for REST API.
Other tests just seems run out of time.... Any thoughts on it?

**Comment by yakimant** [comment]
@NagyZoltanPeter, thanks for fixing my stupid mistakes!

As for `tests/test_waku_enr.nim` - maybe let's change the ENR instead?
I think it would be nice to get rid of wakuv2 references at all.

**Comment by NagyZoltanPeter** [comment]
> @NagyZoltanPeter, thanks for fixing my stupid mistakes!
> 
> As for `tests/test_waku_enr.nim` - maybe let's change the ENR instead? I think it would be nice to get rid of wakuv2 references at all.

Surely, but I did not know what exactly would be the correct expected result, while the test was just about to decode ENR properly.

**Comment by SionoiS** [comment]
> @yakimant, @SionoiS : I pushed fixes for the code and test. Tests seems running well... but not perfect. ubuntu-latest-rln-v2 fails with port is already occupied that test would use for REST API. Other tests just seems run out of time.... Any thoughts on it?

Re-run maybe? Seams like flaky tests except, 1 test in the js interop failed with a shard error.
Maybe some sharding info is wrong in some ENR?

**Pull request merged**